### PR TITLE
Details popup

### PR DESCRIPTION
**Pull request by zkr024**
Add the first popup window to the project. Use an object in javaScript to call the information and also create the HTML tags from JS.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -545,6 +551,106 @@
   outline: none;
 }
 
+/* popup window */
+
+.popup_container {
+  width: 100vw;
```
Comment (ShoiraTa) [inline style.css:557]: - [ ] Please check the mobile view of the details popup, it is not not possible to scroll the popup content down to see buttons. Instead, the body of the page is being scrolled.
Screenshot 2022-03-02 at 19 46 46

File: script.js [hunk 2]
```diff
@@ -23,4 +23,116 @@ closeMenuTwo.addEventListener('click', () => {
...
+});
+
+const cardOne = {
+  name: 'Multi Post Stories',
+  nameTwo: 'Keeping track of hundreds  of components website',
+  technologies: ['html', 'Bootstrap', 'Ruby on rails'],
+  image: 'images/SnapshootPortfolio.png',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+  liveVersion: 'see live',
+  source: 'see source',
```
Comment (ShoiraTa) [inline script.js:35]: One of the requirement for this milestone is 
> The popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins). 

Therefore,  I recommend you changing the background color of these buttons to `#F55800` and adding padding between the text and icon to match with Figma design: 

Your design  | Figma design
------------- | -------------
Screenshot 2022-03-02 at 19 48 46  | Screenshot 2022-03-02 at 19 49 07

File: script.js [hunk 3]
```diff
@@ -23,4 +23,116 @@ closeMenuTwo.addEventListener('click', () => {
...
-});
\ No newline at end of file
+});
+
+const cardOne = {
+  name: 'Multi Post Stories',
+  nameTwo: 'Keeping track of hundreds  of components website',
+  technologies: ['html', 'Bootstrap', 'Ruby on rails'],
+  image: 'images/SnapshootPortfolio.png',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
```
Comment (ShoiraTa) [inline script.js]: - [ ] The JavaScript code should have an array of objects to store JavaScript objects containing the information for each project eg:
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}, 
{
    name: "Next game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
Then you should iterate through projects and load popups dynamically

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi team,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 3 above)
Comment (zkr024) [reply script.js]: Thank you for your feedback @ShoiraTa, I will work on it.

File: script.js [hunk 4]
```diff
@@ -23,4 +23,116 @@ closeMenuTwo.addEventListener('click', () => {
...
+
+const cardOne = [{
+  name: 'Multi Post Stories',
+  nameTwo: 'Keeping track of hundreds  of components website',
+  technologies: ['html', 'Bootstrap', 'Ruby on rails'],
+  image: 'images/SnapshootPortfolio.png',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+  liveVersion: 'see live',
+  source: 'see source',
+}];
```
Comment (keroloussamy) [inline script.js]: - [ ] Kindly as the previous review asked, you here have an array of one object, data for one project, but you should have data for 6-7 projects that's mean you have to have 6-7 different objects everyone has data for a different project.

Like 
```
const projects = [{
    name: "Project one ",
    description: "Project one des",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}, 
{
    name: "Project two",
    description: "Project two des .",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}, 
{
    name: "Project three",
    description: "Project three des .",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}, 
{and the same for project four}, 
{and the same for project five}, 
{and the same for project six}]
```

And use this data to display it in the modal.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @zkr024, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

 ## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 4 above)
Comment (zkr024) [reply script.js]: Thank you @keroloussamy  I will add more objects into the array.

File: style.css (hunk 1 above)
Comment (Kingstalux) [reply style.css:557]: resolved 👍

File: script.js (hunk 2 above)
Comment (Kingstalux) [reply script.js:35]: resolved 👍

File: script.js (hunk 4 above)
Comment (Kingstalux) [reply script.js]: resolved 👍

**Review by Kingstalux — APPROVED** [review]
Hi @zkr024 ,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**